### DISCUSSION TITLE
DAIKIN IR Commands

### DISCUSSION THREAD

**Discussion by fotis3d** [arendst/Tasmota#18177] [Q&A]
Hello all I hope I can get some help here.

I own a Daikin FTX35J2V1B and a tuya IR Blaster.

I flashed the tuya IR with Tasmota IR and I have setup all my devices and they work great. (TV, SoundBar, Toyotomi AC etc)

But I cannot get the Daikin to work. 
I can receive the commands from the remote but when I send them it only beeps once and does not do anything. Except from the power off, that works.

But I cannot get it to turn on .

Here is an example of the received code
MQT: tele/ir_bedroom/RESULT = {"IrReceived":{"Protocol":"DAIKIN","Bits":280,"Data":"0x11DA2700C50000D711DA270042E1023711DA270000493600A0000006600000C18000D8","Repeat":0,"IRHVAC":{"Vendor":"DAIKIN","Model":-1,"Mode":"Heat","Power":"On","Celsius":"On","Temp":27,"FanSpeed":"Auto","SwingV":"Off","SwingH":"Off","Quiet":"Off","Turbo":"Off","Econo":"Off","Light":"Off","Filter":"Off","Clean":"Off","Beep":"Off","Sleep":-1}}}

And here is the send that only beeps once but does not do anything.
IRHVAC":{"Vendor":"DAIKIN","Model":1,"Mode":"Auto","Power":"Off","Celsius":"On","Temp":21,"FanSpeed":"Medium","SwingV":"Off","SwingH":"Off","Quiet":"Off","Turbo":"Off","Econo":"Off","Light":"Off","Filter":"Off","Clean":"Off","Beep":"Off","Sleep":-1}}

If I use the IRsend does not do anything at all.

Any help would be highly appreciated on this. Am I doing something wrong ? or is there anything I do not understand ?

Thanks

**Comment by fotis3d** [comment]
no one ?

**Reply by NsinghP** [reply]
Apologies for Bumping up an old thread, but I think I may have a solution! This problem may be BIT size related. Try using raw data in your command string. First turn Setoption58 to 1, then capture raw data from the console and then send the IRsend command with Raw data. See my post [#20958](https://github.com/arendst/Tasmota/discussions/20958) for how it worked in my case. My air-conditioner uses 112 Bits for IRCommands.
